guides/common/modules/proc_configuring-repositories.adoc

```diff
@@ -28,32 +41,61 @@ ifdef::foreman-el,katello[]
 
 include::proc_configuring-repositories-el.adoc[]
 
-ifdef::katello[]
+include::snip_step-verify-enabled-repolist.adoc[]
+
 +
-. Enable `powertools` repository:
+. Enable the DNF module:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-manager} config-manager --set-enabled powertools
+# dnf module enable {dnf-module}
 ----
+
 endif::[]
 
+ifdef::foreman-el,katello,satellite[]
+[id="repositories-el-9"]
+== {EL} 9
+
+:distribution: rhel
+:distribution-major-version: 9
+:package-manager: dnf
+
+. Disable all repositories:
 +
-. Enable the DNF modules:
+[options="nowrap"]
+----
+# subscription-manager repos --disable "*"
+----
++
+. Enable the following repositories:
 +
 [options="nowrap" subs="+quotes,attributes"]
+ifdef::foreman-el,katello[]
 ----
-# dnf module enable {dnf-module}
+# subscription-manager repos \
+--enable={RepoRHEL9BaseOS} \
+--enable={RepoRHEL9AppStream}
+----
+endif::[]
+ifdef::satellite[]
+----
+# subscription-manager repos \
+--enable={RepoRHEL9BaseOS} \
+--enable={RepoRHEL9AppStream} \
+--enable={RepoRHEL9ServerSatelliteServerProjectVersion} \
+--enable={RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
 ----
-
 endif::[]
 
 ifdef::foreman-el,katello[]
-[id="repositories-rhel-8"]
-== {RHEL} 8
+include::proc_configuring-repositories-el.adoc[]
 endif::[]
 
-ifdef::foreman-el,katello,satellite[]
+include::snip_step-verify-enabled-repolist.adoc[]
+
+[id="repositories-el-8"]
+== {EL} 8
```

Comment:
This is true RHEL (like it already was)
```suggestion
[id="repositories-rhel-8"]
== {RHEL} 8
```

Reply:
The repos apply to other builds as well and that was the reason we used {EL}. I had the same discussion with @maximiliankolb.

Reply:
After testing this in other EL, I found that `subscription-manager` does not apply there and so, it is exclusive to RHEL when it comes to this procedure. 

Other EL needs to go via `dnf config-manager` path for disabling and enabling repos. Those systems have no repositories available through subscription-manager (CLI reads).

@maximiliankolb we may need to reconsider this - https://github.com/theforeman/foreman-documentation/pull/2805#discussion_r1505825181. I agree with Ewoud here, this procedure is exclusive to RHEL

Reply:
Yes, this section only applies to RHEL (technically you could also run Foreman/Katello on RHEL ...) -> see https://github.com/theforeman/foreman-documentation/pull/2805/files#r1513973554

Reply:
@ekohl I have classified EL and RHEL repos. I have made sure RHEL commands do not get included outside Satellite.

Reply:
The instructions for CentOS Stream and EL are now exactly the same, so it's sufficient to only list EL. This is now possible because Katello no longer needs the powertools repository, which has a different name on RHEL.

Reply:
Totally agree. Delisted CentOS :+1: